provider/awsprovisioner/metadata_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"net/http/httptest"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	assert "github.com/stretchr/testify/require"
```

Comment:
In the examples, this is imported as `require`, not `assert` -- probably the same should be true here?